lib/jsanalyze.js

```diff
@@ -190,7 +192,11 @@ exports.analyzeJs = function analyzeJs(contents, opts) {
 		options.plugins.push(require.resolve('babel-plugin-transform-property-literals'));
 	}
 
-	if (options.presets.length) {
+	if (opts.plugins.length) {
```

Comment:
won't this fail if the user doesn't pass in a plugins property on the options object? I think you need to set a default values of `[]` if not specified.